TechDraw/LasercutterSVGExport.FCMacro

```diff
@@ -6,14 +6,14 @@ __Author__ = 'christi'
 __Version__ = '0.4'
 __Date__ = '2019-07-14'
 __License__ = 'LGPL-3.0-or-later'
-__Web__ = ''
-__Wiki__ = ''
+__Web__ = 'https://github.com/FreeCAD/FreeCAD-macros/tree/master/TechDraw'
+__Wiki__ = 'https://github.com/FreeCAD/FreeCAD-macros/tree/master/TechDraw/LasercutterSVGExport'
 __Icon__ = ''
 __Help__ = 'LasercutterSVGExport/Readme.md'
 __Status__ = 'Alpha'
 __Requires__ = 'FreeCAD >= v0.18'
 __Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=15&t=31796'
-__Files__ = 'LasercutterSVGExport/Readme.md,LasercutterSVGExport/LasercutterSVGExport_screenshot.png'
+__Files__ = 'LasercutterSVGExport/Readme.md,LasercutterSVGExport/LasercutterSVGExport_screenshot.png,LasercutterSVGExport/lasercuttersvg.ui'
```

Comment:
We don't need the `ui` file at runtime for now, so that I wouldn't include it here. My plan is to have macro programmers use `ui` files at runtime rather than using a script where part of the code is generated. So far, no macro does this and I may use @chbergmann 's macro to show how to do it (or anybody would wants to do it) and that's why I asked for it.

Reply:
How does it work to use a .ui file in a macro ?